lib/contants/routes.dart

```diff
@@ -34,7 +34,7 @@ class AppRoutes {
       mainPage: (context) => const MainPageView(),
       units: (context) => const UnitsView(),
       words: (context) => const WordsView(),
-      teachWord: (context) => const TeachWordView(),
+      // teachWord: (context) => const TeachWordView(),
```

Comment:
Why is this line commented out?

Reply:
I use Navigator.of(context).push(MaterialPageRoute()) instead, cause i need to get the arg in initState, not in build

Reply:
does it make sense?🤔 or are there any alternative method?